framework/src/mesh/MooseMesh.C

```diff
@@ -358,33 +354,28 @@ MooseMesh::freeBndElems()
 }
 
 void
-MooseMesh::prepare(bool force)
+MooseMesh::prepare(bool)
 {
   TIME_SECTION(_prepare_timer);
 
   mooseAssert(_mesh, "The MeshBase has not been constructed");
 
-  if (dynamic_cast<DistributedMesh *>(&getMesh()) && !_is_nemesis)
-  {
-    // Call prepare_for_use() and don't mess with the renumbering
-    // setting
-    if (force || _needs_prepare_for_use)
-    {
-      CONSOLE_TIMED_PRINT("Preparing for use");
+  if (!dynamic_cast<DistributedMesh *>(&getMesh()) || _is_nemesis)
+    // For whatever reason we do not want to allow renumbering here nor ever in the future?
```

Comment:
Any idea on what the logic is here?

I could understand it in some specific cases in testing... but that's about it. The more the contiguous the numbering the better, no?